airflow/contrib/example_dags/example_kubernetes_operator.py

```diff
@@ -63,6 +99,24 @@
             tolerations=tolerations
         )
 
+        start_pod = KubernetesPodYamlOperator(
```

Comment:
Do we need a separate operator for this? Might a `KubeernetesPodOperator(yaml=...)` work?

Reply:
I am afraid that it will be very problematic to combine these two ways of defining pods. All we can do is say that the selected parameters are mutually exclusive.  However, it will be very difficult for the user to understand if which parameters work with each other and which do not work.  A similar case is in the configuration of the worker, where when configuring the template it is said that all other fields are ignored.  However, In this case, we will ignore all parameters, but a very long list of parameters that will be difficult to use.

Reply:
The following parameters from KubernetesPodOperator do not work with this operator:
affinity
annotations
args
cmds
configmaps
dnspolicy
envs
hostnetwork
image
image_pull_policy
image_pull_secrets
labels
name
namespace
node_selectors
pod
resources
security_context
service_account_name
tolerations